recipe/bld.bat

```diff
@@ -1,7 +0,0 @@
-set SITECFG=cf_units/etc/site.cfg
-echo [System] > %SITECFG%
-echo udunits2_path = %SCRIPTS%\udunits2.dll >> %SITECFG%
-echo udunits2_xml_path = %LIBRARY_PREFIX%\share\udunits\udunits2.xml >> %SITECFG%
```

Comment:
Personally, I'd have kept this, as it helps cf_units work even if users don't activate their environment.
That said, I'm not overly concerned about it 😄

Reply:
Makes sense. I can re-add that in another PR later. I guess I was confused due to your comment on this line for Linux and OS X before. Should I re-add `build.sh` and `bld.bat` with the `cf_units/etc/site.cfg` and the `udunits2.xml` path on both?

Reply:
Sorry, confusing messages. conda does the binary replacement just fine for OSX and Linux, just not for Windows. Happy to have the config file for all platforms though for consistency.

Reply:
No problem.